lib/utils.js

```diff
@@ -1,6 +1,18 @@
 import NativePromise from 'native-promise-only'
+import pkgJson from '../package.json'
+
 const Promise = window.Promise || NativePromise
 
+export function addUserAgent (headers, appId) {
+  const originalUserAgent = headers && headers['User-Agent'] ? headers['User-Agent'] : ''
+  const zafSdkUserAgentString = `zendesk_app_framework_sdk/sdk_version:${pkgJson.version}/app_id:${appId}`
```

Comment:
I reckon the version in pkgJson doesn't reflect the actual version, this PR didn't touch on the version as well

Reply:
Thank you Sheen, should I remove this? Note that I've bumped the version in package.json [in this commit](https://github.com/zendesk/zendesk_app_framework_sdk/pull/220/commits/23e6df883bfab63cfa495c7c8bed55be8b603c91) but I can remove if you prefer.

Reply:
I reckon when customer refer to zaf_sdk in their app, it is using the [release version](https://github.com/zendesk/zendesk_app_framework_sdk/releases) now. So this version is unrelavant unless we can manually keep align with the release version. 

With regarding to shall we keep this, as reviewing the purpose of this PR, wonder do you need this info for internal debug only? In that case, probably considering adding x-zendesk-app-framework-sdk-version header for this instead of chain the existing user agent? As all `x-zendesk-` headers will be filtered(unless in allowlist) out before zendesk proxy dispatch the request, while in the meantime, internal can still track the zaf-sdk version in the log